packages/lodestar-beacon-state-transition/src/phase0/fast/epoch/processRewardsAndPenalties.ts

```diff
@@ -18,14 +18,14 @@ export function processRewardsAndPenalties(
   const newBalances = readOnlyMap(state.balances, (balance) => balance);
```

Comment:
As an additional optimization, you might try to apply the rewards and penalties at the same time.
eg:
```ts
const newBalances = readOnlyMap(state.balances, (balance, i) => {
  const newBalance = balance + BigInt(rewards[i] - penalties[i]);
  return newBalance < 0 ? BigInt(0) : newBalance;
});
```

Reply:
@wemeetagain this helps reduce the time in `processRewardsAndPenalties.test.ts` ~5%